### PR TITLE
Updated nl.js based on en.js

### PR DESCRIPTION
**Pull request by ShakeAndWiggle20**
I found some untranslated sentences and words, so i translated them for you.

### CODE REVIEW

**Comment by ShakeAndWiggle20** [comment]
Please click on "files changed" and review my changes.

**Comment by mty22** [comment]
@nickauthentic can you have a squiz please? :)

File: nl.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ generic.password Wachtwoord Invoeren
 generic.register Registreren
 generic.login Inloggen
 generic.accuracy Nauwkeurigheid
-player.waiting wachten op spelers
+player.waiting Wachten op spelers
```
Comment (nickauthentic) [inline nl.js:61]: hoeft niet per se een hoofdletter te hebben

**Review by nickauthentic — APPROVED** [review]
The rest looks good! Thanks a lot

File: nl.js (hunk 1 above)
Comment (mty22) [reply nl.js:61]: ill fix it post merge

**Comment by mty22** [comment]
@ShakeAndWiggle20 please let know your IGN by hitting me up in KB or reply here (ping me)

File: nl.js (hunk 1 above)
Comment (mty22) [reply nl.js:61]: fixed: https://github.com/mty22/krunker.io-translations/commit/b08d59bfb12a7f6a3fb7e6d37b36bcbf490e92dc

**Comment by ShakeAndWiggle20** [comment]
> @ShakeAndWiggle20 please let know your IGN by hitting me up in KB or reply here (ping me)

My ingame name is "20Th0masQ"

**Review by trovie23 — COMMENTED** [review]
Heb een paar dingen veranderd, maar ziet er goed uit tot nu toe.

**Pull request merged**